docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "2.4"
+
+services:
+  phoxicontrol:
+    image: photoneo/phoxicontrol
+    runtime: nvidia
```

Comment:
Btw. You don't have to use Nvidia. So you can use your integrated graphics card
```suggestion
    devices:
      - /dev/dri:/dev/dri
```

Reply:
Do we still need opengl base image?